.github/workflows/e2e-test.yaml

```diff
@@ -40,7 +40,15 @@ jobs:
         run: |
           go get -v -t -d ./... && go mod tidy
           echo ::set-output name=exit_code::$?
-      
+
```

Comment:
Do we need to add this in e2e tests? We already have in unit test workflow and we can skip here.